Makefile

```
@@ -16,6 +16,8 @@ CHARTS_INTEGRATION    := wire-server databases-ephemeral fake-aws nginx-ingress-
 # CHARTS_RELEASE := $(shell find charts/ -maxdepth 1 -type d | xargs -n 1 basename | grep -v charts)
 CHARTS_RELEASE        := wire-server databases-ephemeral fake-aws aws-ingress backoffice calling-test demo-smtp elasticsearch-curator elasticsearch-external fluent-bit minio-external cassandra-external nginx-ingress-controller nginx-ingress-services reaper wire-server-metrics sftd
 BUILDAH_PUSH          ?= 1
```

This PR seems to imply a local workflow, so pushing images to remote seems like not the right default:

```suggestion
BUILDAH_PUSH          ?= 0
```

---

Hmm, I didn't want to break somebody's workflow. But I guess it is just you and me who use this as of now, so I will break it and also enable BUILDAH_KIND_LOAD.

---

Done.